dbms/src/Flash/Mpp/MPPTaskManager.cpp

```diff
@@ -89,31 +97,22 @@ void MPPTaskManager::cancelMPPQuery(UInt64 query_id, const String & reason)
     LOG_WARNING(log, fmt::format("Begin cancel query: {}", query_id));
     FmtBuffer fmt_buf;
     fmt_buf.fmtAppend("Remaining task in query {} are: ", query_id);
-    // TODO: cancel tasks in order rather than issuing so many threads to cancel tasks
```

Comment:
It looks like this annotation cannot be removed, or not using `thread_manager`

Reply:
Add it back